src/jemalloc.c

```diff
@@ -1052,9 +1050,6 @@ malloc_conf_init(void) {
 
 			CONF_HANDLE_BOOL(opt_abort, "abort")
 			CONF_HANDLE_BOOL(opt_abort_conf, "abort_conf")
-			if (opt_abort_conf && had_conf_error) {
```

Comment:
Previously we may never reach this point, since CONF_HANDLE does `continue` when matches.